lib/services/ios-debug-service.ts

```diff
@@ -197,7 +197,13 @@ class IOSDebugService extends DebugServiceBase implements IPlatformDebugService
 			this._socketProxy = await this.$socketProxyFactory.createWebSocketProxy(this.getSocketFactory(device));
 
 			const commitSHA = "02e6bde1bbe34e43b309d4ef774b1168d25fd024"; // corresponds to 55.0.2883 Chrome version
-			return `chrome-devtools://devtools/remote/serve_file/@${commitSHA}/inspector.html?experiments=true&ws=localhost:${this._socketProxy.options.port}`;
+			let chromeDevToolsPrefix = `chrome-devtools://devtools/remote/serve_file/@${commitSHA}`;
```

Comment:
You can extract `chrome-devtools://devtools/` to a constant. Not a merge-stopper though